test/e2e/lua/dynamic_configuration.go

```diff
@@ -290,8 +293,6 @@ func enableDynamicConfiguration(namespace string, kubeClientSet kubernetes.Inter
 				return err
 			}
 
-			time.Sleep(5 * time.Second)
-
```

Comment:
this was probably some leftover from previous implementation - I don't see any reason for it to stay now.